Doc/naming-conventions.md

```diff
@@ -0,0 +1,88 @@
+# Metric and tag naming convention for M6 usage
+
+## 1. Metric naming convention
+
+### Follow Prometheus guideline
+
+A metric should respect this format:
+```
+{source}_{subject/type}_({direction:in/out}_)_{unit}
+```
+
+1. A __source__ prefix: `http`, `worker`, `command`, etc.
+2. An explicit __subject__: `request`, `execution_duration`
+3. A __direction__: does the metric monitore something coming `in`,  `out` or is internal?
+4. And a __unit__ suffix: 
+    * increment => `_total`
+    * counter => `_count`
+    * duration => `_seconds`
+    * memory => `_bytes`
+    * etc.
+
+Please, have a look at the [Prometheus metric and label naming convention](https://prometheus.io/docs/practices/naming/).
+
+### Allowed characters and format
+
+In order to name the metric, you can use lower case letters, numbers and underscores: __[a-z0-9\_]__.
+
+You are recommended to name the metrics in __snake case__. 
+
+The statsd_exporter will transform every of metric names in snake_case anyway. 
+It will be easier for you to find your metrics later if the names are the same. 
+
+
+### Unity
+
+Metrics name must be __unique__. 
+
+You cannot have the same metric name, with a different type or differents tag names, for example.
+
+The duplicated metric will be __ignored__.
+ 
+### Stop using dynamic values 
+
+Placeholders (dynamic variables like __request.\<statusCode\>__) in metric names are now __deprecated__. 
+
+Please, use __tags__ instead (See [6. Configure the tags](configuration.md#6-configure-the-tags)). 
+
+
+## 2. Tags naming convention
+
+In order to have common metric names amongst all of our applications, we have defined some common tags names.
+If you need to get one of this data, please use the following name for your tag.
+
+#### Global context
+ 
+* `project`: __\[required\]__ 
+Your project name. *(service_6play_users_cloud, service_6play_middleware, ...)*
+* `customer`: default value : *default*. Can be *all* for non-related customer applications.
+* :warning: __Waiting for validation__ `release`: Value matching with the header X-Client-Release: 
+This will create TOO MUCH metrics 
+
+
+:fire_engine: @Todo: the bundle has to bu updated to allow dynamic values into configuration tags.
+
+
+#### Group context
+
+None yet.
```

Comment:
Does that mean that there is no naming convention yet?

Reply:
Absolutely. I'll explicit this.